mcp-server-gitparser/src/mcp_server_gitparser/config.py

```
@@ -0,0 +1,57 @@
+    model_config = SettingsConfigDict(
+        env_file=_env_file,
+        env_file_encoding="utf-8",
+        env_prefix="MCP_GITPARSER_",
+        env_nested_delimiter="__",
+        case_sensitive=False,
+        extra="ignore",
+    )
+
+    @computed_field
+    @property
+    def project_root(self) -> Path:
```

Ruff (0.14.14)

[warning] Missing docstring in public method

(D102)